docs/tutorial.rst

```diff
@@ -3,6 +3,96 @@
 ========
 Tutorial
 ========
+There are two main ways of accessing the resources of the ``stdpopsim`` package that will
+be detailed in this tutorial. The first is via the command line interface (CLI). This is
+useful if you want to do a straightforward run of the models in the ``stdpopsim``
+catalog. The other way to access the ``stdpopsim`` resources is via the python API. This
+is a bit more complicated however it allows for more advanced tasks. This tutorial will
+walk through both ways of using the ``stdpopsim`` package as well as some more advanced
+tasks you may wish to do.
+
+***************************
+Using the ``stdpopsim`` CLI
+***************************
+The CLI provides access to the catalog of models that have already been implemented by
+``stdpopsim``. The first step for using the CLI is to select the species that you are
+interested in simulating data for. In order to see which species are available run
+
+.. code-block:: console
+
+    $ stdpopsim --help
+
+This shows the species currently supported by ``stdpopsim``. This means that
+``stdpopsim`` knows various traits of these species including chromosome size and
+recombination rates. Once we've selected a species, in this case humans, we can look at
+the help again as follows.
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help
+
+For conciseness we do not show the output here but this time you should see a different
+output which shows options for performing the simulation itself and the species default
+parameters. This includes selecting the demographic model, chromosome, recombination map,
+and number of samples. The most basic simulation we can run is to draw two samples using
```

Comment:
Add a paragraph space before "The most basic simulation..."

Reply:
> Longer night than expected - I'm a bit tired, will look first thing tomorrow. My comments are p minor compared to Ariella's so if there is another pull request about this page, I can just add them there

I'm working on hers so just add whatever you have into the thread and I'll try to get to them promptly.